cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -135,6 +155,14 @@ var eventTypeFromXDR = map[xdr.ContractEventType]string{
 	xdr.ContractEventTypeDiagnostic: EventTypeDiagnostic,
 }
 
+func getEventTypeXDRFromEventType() map[string]xdr.ContractEventType {
+	return map[string]xdr.ContractEventType{
+		EventTypeSystem:     xdr.ContractEventTypeSystem,
+		EventTypeContract:   xdr.ContractEventTypeContract,
+		EventTypeDiagnostic: xdr.ContractEventTypeDiagnostic,
+	}
+}
+
```

Comment:
Can/Should this just be a `var` at the top of the file? You could then use `len(EventTypeMap)` instead of a hardcoded `3` constant elsewhere.

Reply:
well initially it was `var` same as `eventTypeFromXDR` but linter does not like global var :( so it was more of temp fix. it will all be gone when we target all of this lint error together to have consistency in code.